go.mod

```diff
@@ -1,212 +1,107 @@
 module go.mondoo.com/packer-plugin-cnspec
 
-go 1.19
+go 1.21
+
+toolchain go1.21.1
 
 replace github.com/slack-go/slack v0.12.1 => github.com/imilchev/slack v0.0.0-20230324120548-5380d7dd00a5
 
+require go.mondoo.com/cnquery/v9 v9.0.2
+
+require go.mondoo.com/cnspec/v9 v9.0.2
+
 require (
-	github.com/cockroachdb/errors v1.9.1
+	github.com/cockroachdb/errors v1.11.1
 	github.com/hashicorp/hcl/v2 v2.14.1
```

Comment:
nit: when we update those anyway we should use the opportunity to get to latest and greatest

Reply:
Done